testnet/stacks-node/Cargo.toml

```diff
@@ -23,6 +23,7 @@ libc = "0.2"
 slog = { version = "2.5.2", features = [ "max_level_trace" ] }
 
 [dev-dependencies]
+ring = "0.16.19"
```

Comment:
Are we using this anywhere?

Reply:
ha the ring thing is already pulled by another crate, I'm just specifying a version to get this build working on Apple Silicon :)